marketplace_docs/templates/Fabric/files/etc/update-motd.d/99-one-click

```diff
@@ -2,14 +2,12 @@
 #
 # Configured as part of the DigitalOcean 1-Click Image build process
 
-myip=$(hostname -I | awk '{print$1}')
 cat <<EOF
 ********************************************************************************
 
-This is the text that will be shown in the MOTD users see when they log in via
-SSH.  This copy should include links to documentation or product informaiton
-along with any information that would helpful to users.
+If you have any queries visit https://chat.zulip.org to get help from the Zulip community.
```

Comment:
We should probably link to the ReadTheDocs troubleshooting guide, since that has some basic guidelines like that they should go to `#production help`.